06-06 16:02:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
create table Ads (_id bigint generated by default as identity, id bigint not null, dateFrom timestamp, dateTo timestamp, description varchar(255), name varchar(255), publisher__id bigint, publisher_id bigint, primary key (_id, id)) [90017-166]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:236)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:82)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5192)
	at org.h2.command.Parser.parseCreate(Parser.java:3806)
	at org.h2.command.Parser.parsePrepared(Parser.java:324)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:217)
	at org.h2.engine.Session.prepareLocal(Session.java:415)
	at org.h2.engine.Session.prepareCommand(Session.java:364)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:121)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:110)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeUpdate(NewProxyStatement.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(SchemaUpdate.java:204)
	at org.hibernate.impl.SessionFactoryImpl.<init>(SessionFactoryImpl.java:375)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1872)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:906)
	at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:267)
	at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:525)
	at play.Play.start(Play.java:533)
	at play.Play.detectChanges(Play.java:649)
	at play.Invoker$Invocation.init(Invoker.java:198)
	at play.server.PlayHandler$NettyInvocation.init(PlayHandler.java:190)
	at play.Invoker$Invocation.run(Invoker.java:276)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:230)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
06-06 16:02:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
create table Subscription (_id bigint generated by default as identity, id bigint not null, shortDescription varchar(255), ad__id bigint, ad_id bigint, subscribedUser__id bigint, subscribedUser_id bigint, primary key (_id, id)) [90017-166]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:236)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:82)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5192)
	at org.h2.command.Parser.parseCreate(Parser.java:3806)
	at org.h2.command.Parser.parsePrepared(Parser.java:324)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:217)
	at org.h2.engine.Session.prepareLocal(Session.java:415)
	at org.h2.engine.Session.prepareCommand(Session.java:364)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:121)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:110)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeUpdate(NewProxyStatement.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(SchemaUpdate.java:204)
	at org.hibernate.impl.SessionFactoryImpl.<init>(SessionFactoryImpl.java:375)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1872)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:906)
	at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:267)
	at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:525)
	at play.Play.start(Play.java:533)
	at play.Play.detectChanges(Play.java:649)
	at play.Invoker$Invocation.init(Invoker.java:198)
	at play.server.PlayHandler$NettyInvocation.init(PlayHandler.java:190)
	at play.Invoker$Invocation.run(Invoker.java:276)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:230)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
06-06 16:02:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
create table Users (_id bigint generated by default as identity, id bigint not null, name varchar(255), password varchar(255), phone varchar(255), role integer not null, username varchar(255), primary key (_id, id)) [90017-166]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:236)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:82)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5192)
	at org.h2.command.Parser.parseCreate(Parser.java:3806)
	at org.h2.command.Parser.parsePrepared(Parser.java:324)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:217)
	at org.h2.engine.Session.prepareLocal(Session.java:415)
	at org.h2.engine.Session.prepareCommand(Session.java:364)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:121)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:110)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeUpdate(NewProxyStatement.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(SchemaUpdate.java:204)
	at org.hibernate.impl.SessionFactoryImpl.<init>(SessionFactoryImpl.java:375)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1872)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:906)
	at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:267)
	at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:525)
	at play.Play.start(Play.java:533)
	at play.Play.detectChanges(Play.java:649)
	at play.Invoker$Invocation.init(Invoker.java:198)
	at play.server.PlayHandler$NettyInvocation.init(PlayHandler.java:190)
	at play.Invoker$Invocation.run(Invoker.java:276)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:230)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
06-06 16:02:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "ADS" not found; SQL statement:
alter table Ads add constraint FK10090D322A099 foreign key (publisher__id, publisher_id) references Users [42102-166]
06-06 16:02:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "SUBSCRIPTION" not found; SQL statement:
alter table Subscription add constraint FK1E21AD3D2F2BC161 foreign key (ad__id, ad_id) references Ads [42102-166]
06-06 16:02:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "SUBSCRIPTION" not found; SQL statement:
alter table Subscription add constraint FK1E21AD3DFAB7E779 foreign key (subscribedUser__id, subscribedUser_id) references Users [42102-166]
06-06 16:03:52 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "USERS" not found; SQL statement:
select users0_._id as column1_2_, users0_.id as id2_, users0_.name as name2_, users0_.password as password2_, users0_.phone as phone2_, users0_.role as role2_, users0_.username as username2_ from Users users0_ [42102-166]
06-06 16:04:50 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
create table Ads (_id bigint generated by default as identity, id bigint not null, dateFrom timestamp, dateTo timestamp, description varchar(255), name varchar(255), publisher__id bigint, publisher_id bigint, primary key (_id, id)) [90017-166]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:236)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:82)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5192)
	at org.h2.command.Parser.parseCreate(Parser.java:3806)
	at org.h2.command.Parser.parsePrepared(Parser.java:324)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:217)
	at org.h2.engine.Session.prepareLocal(Session.java:415)
	at org.h2.engine.Session.prepareCommand(Session.java:364)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:121)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:110)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeUpdate(NewProxyStatement.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(SchemaUpdate.java:204)
	at org.hibernate.impl.SessionFactoryImpl.<init>(SessionFactoryImpl.java:375)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1872)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:906)
	at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:267)
	at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:525)
	at play.Play.start(Play.java:533)
	at play.Play.detectChanges(Play.java:637)
	at play.Invoker$Invocation.init(Invoker.java:198)
	at play.server.PlayHandler$NettyInvocation.init(PlayHandler.java:190)
	at play.Invoker$Invocation.run(Invoker.java:276)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:230)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
06-06 16:04:50 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
create table Subscription (_id bigint generated by default as identity, id bigint not null, shortDescription varchar(255), ad__id bigint, ad_id bigint, subscribedUser__id bigint, subscribedUser_id bigint, primary key (_id, id)) [90017-166]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:236)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:82)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5192)
	at org.h2.command.Parser.parseCreate(Parser.java:3806)
	at org.h2.command.Parser.parsePrepared(Parser.java:324)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:217)
	at org.h2.engine.Session.prepareLocal(Session.java:415)
	at org.h2.engine.Session.prepareCommand(Session.java:364)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:121)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:110)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeUpdate(NewProxyStatement.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(SchemaUpdate.java:204)
	at org.hibernate.impl.SessionFactoryImpl.<init>(SessionFactoryImpl.java:375)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1872)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:906)
	at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:267)
	at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:525)
	at play.Play.start(Play.java:533)
	at play.Play.detectChanges(Play.java:637)
	at play.Invoker$Invocation.init(Invoker.java:198)
	at play.server.PlayHandler$NettyInvocation.init(PlayHandler.java:190)
	at play.Invoker$Invocation.run(Invoker.java:276)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:230)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
06-06 16:04:50 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
create table Users (_id bigint generated by default as identity, id bigint not null, name varchar(255), password varchar(255), phone varchar(255), role integer not null, username varchar(255), primary key (_id, id)) [90017-166]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:236)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:82)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5192)
	at org.h2.command.Parser.parseCreate(Parser.java:3806)
	at org.h2.command.Parser.parsePrepared(Parser.java:324)
	at org.h2.command.Parser.parse(Parser.java:279)
	at org.h2.command.Parser.parse(Parser.java:251)
	at org.h2.command.Parser.prepareCommand(Parser.java:217)
	at org.h2.engine.Session.prepareLocal(Session.java:415)
	at org.h2.engine.Session.prepareCommand(Session.java:364)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1111)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:121)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:110)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeUpdate(NewProxyStatement.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(SchemaUpdate.java:204)
	at org.hibernate.impl.SessionFactoryImpl.<init>(SessionFactoryImpl.java:375)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1872)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:906)
	at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:267)
	at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:525)
	at play.Play.start(Play.java:533)
	at play.Play.detectChanges(Play.java:637)
	at play.Invoker$Invocation.init(Invoker.java:198)
	at play.server.PlayHandler$NettyInvocation.init(PlayHandler.java:190)
	at play.Invoker$Invocation.run(Invoker.java:276)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:230)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
06-06 16:04:50 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "ADS" not found; SQL statement:
alter table Ads add constraint FK10090D322A099 foreign key (publisher__id, publisher_id) references Users [42102-166]
06-06 16:04:50 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SUBSCRIPTION" not found; SQL statement:
alter table Subscription add constraint FK1E21AD3D2F2BC161 foreign key (ad__id, ad_id) references Ads [42102-166]
06-06 16:04:50 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "SUBSCRIPTION" not found; SQL statement:
alter table Subscription add constraint FK1E21AD3DFAB7E779 foreign key (subscribedUser__id, subscribedUser_id) references Users [42102-166]
06-06 16:04:51 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "USERS" not found; SQL statement:
select users0_._id as column1_2_, users0_.id as id2_, users0_.name as name2_, users0_.password as password2_, users0_.phone as phone2_, users0_.role as role2_, users0_.username as username2_ from Users users0_ [42102-166]
06-09 14:39:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
alter table Ads add constraint FK10090D322A099 foreign key (publisher__id, publisher_id) references Users [21002-166]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:199)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:68)
	at org.h2.command.CommandContainer.update(CommandContainer.java:73)
	at org.h2.command.Command.executeUpdate(Command.java:226)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:125)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:110)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeUpdate(NewProxyStatement.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(SchemaUpdate.java:204)
	at org.hibernate.impl.SessionFactoryImpl.<init>(SessionFactoryImpl.java:375)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1872)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:906)
	at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:267)
	at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:525)
	at play.Play.start(Play.java:533)
	at play.Play.detectChanges(Play.java:649)
	at play.Invoker$Invocation.init(Invoker.java:198)
	at play.server.PlayHandler$NettyInvocation.init(PlayHandler.java:190)
	at play.Invoker$Invocation.run(Invoker.java:276)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:230)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
06-09 14:39:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
alter table Subscription add constraint FK1E21AD3DFAB7E779 foreign key (subscribedUser__id, subscribedUser_id) references Users [21002-166]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:199)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:68)
	at org.h2.command.CommandContainer.update(CommandContainer.java:73)
	at org.h2.command.Command.executeUpdate(Command.java:226)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:125)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:110)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeUpdate(NewProxyStatement.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(SchemaUpdate.java:204)
	at org.hibernate.impl.SessionFactoryImpl.<init>(SessionFactoryImpl.java:375)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1872)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:906)
	at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:267)
	at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:525)
	at play.Play.start(Play.java:533)
	at play.Play.detectChanges(Play.java:649)
	at play.Invoker$Invocation.init(Invoker.java:198)
	at play.server.PlayHandler$NettyInvocation.init(PlayHandler.java:190)
	at play.Invoker$Invocation.run(Invoker.java:276)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:230)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
06-09 14:40:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
alter table Ads add constraint FK10090D322A099 foreign key (publisher__id, publisher_id) references Users [21002-166]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:199)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:68)
	at org.h2.command.CommandContainer.update(CommandContainer.java:73)
	at org.h2.command.Command.executeUpdate(Command.java:226)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:125)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:110)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeUpdate(NewProxyStatement.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(SchemaUpdate.java:204)
	at org.hibernate.impl.SessionFactoryImpl.<init>(SessionFactoryImpl.java:375)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1872)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:906)
	at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:267)
	at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:525)
	at play.Play.start(Play.java:533)
	at play.Play.detectChanges(Play.java:649)
	at play.Invoker$Invocation.init(Invoker.java:198)
	at play.server.PlayHandler$NettyInvocation.init(PlayHandler.java:190)
	at play.Invoker$Invocation.run(Invoker.java:276)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:230)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
06-09 14:40:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
alter table Subscription add constraint FK1E21AD3DFAB7E779 foreign key (subscribedUser__id, subscribedUser_id) references Users [21002-166]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:199)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:68)
	at org.h2.command.CommandContainer.update(CommandContainer.java:73)
	at org.h2.command.Command.executeUpdate(Command.java:226)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:125)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:110)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeUpdate(NewProxyStatement.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaUpdate.execute(SchemaUpdate.java:204)
	at org.hibernate.impl.SessionFactoryImpl.<init>(SessionFactoryImpl.java:375)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1872)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:906)
	at play.db.jpa.JPAPlugin.onApplicationStart(JPAPlugin.java:267)
	at play.plugins.PluginCollection.onApplicationStart(PluginCollection.java:525)
	at play.Play.start(Play.java:533)
	at play.Play.detectChanges(Play.java:649)
	at play.Invoker$Invocation.init(Invoker.java:198)
	at play.server.PlayHandler$NettyInvocation.init(PlayHandler.java:190)
	at play.Invoker$Invocation.run(Invoker.java:276)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:230)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
07-31 13:25:43 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-166]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1386)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1361)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:423)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.getAutoCommit(NewProxyConnection.java:985)
	at org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:81)
	at org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1473)
	at org.hibernate.ejb.TransactionImpl.begin(TransactionImpl.java:60)
	at play.db.jpa.JPAPlugin.startTx(JPAPlugin.java:377)
	at play.db.jpa.JPAPlugin.beforeInvocation(JPAPlugin.java:345)
	at play.plugins.PluginCollection.beforeInvocation(PluginCollection.java:473)
	at play.Invoker$Invocation.before(Invoker.java:217)
	at play.Invoker$Invocation.run(Invoker.java:277)
	at play.server.PlayHandler$NettyInvocation.run(PlayHandler.java:230)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
